sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample1_HelloWorld.md

```diff
@@ -107,4 +107,4 @@ await client.PurgeDeletedKeyAsync(rsaKeyName);
 * [Synchronous Sample1_HelloWorld.cs](../tests/samples/Sample1_HelloWorld.cs)
 * [Asynchronous Sample1_HelloWorldAsync.cs](../tests/samples/Sample1_HelloWorldAsync.cs)
 
-[DefaultAzureCredential]: ../../../identity/Azure.Identity/README.md
+[DefaultAzureCredential]: ../../../identity/Azure.Identity/README.md/#DefaultAzureCredential
```

Comment:
1. The slash before the bookmark is invalid. I'm surprised this even works, but README.md is not a folder.
2. I don't agree with this change. For starters, you're assuming that this bookmark in a completely separate library will remain the same. It's creates a tight coupling for which there's no guarantees it will continue to work. The README also covers more information that is important to understand what DefaultAzureCredential does, IMO.

Reply:
Please change all instances (or, rather, unchange them). I'll not repeat this comment.